Documentation/guides/k8s-aws-ansible/install.rst

```diff
@@ -0,0 +1,1430 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium$ mkdir installation;cd installation/
+    core@host:~/cilium$ aws ec2 create-vpc --cidr-block 192.168.32.0/19
+    core@host:~/cilium/installation$ export VPC_ID=vpc-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 create-subnet --vpc-id $VPC_ID --cidr-block 192.168.32.0/19
+    core@host:~/cilium/installation$ export SUBNET_ID=subnet-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 modify-subnet-attribute --subnet-id $SUBNET_ID --map-public-ip-on-launch
+    core@host:~/cilium/installation$ aws ec2 create-internet-gateway
+    core@host:~/cilium/installation$ export GATEWAY_ID=igw-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 attach-internet-gateway --vpc-id $VPC_ID --internet-gateway $GATEWAY_ID
+    core@host:~/cilium/installation$ aws ec2 create-route-table --vpc-id $VPC_ID
+    core@host:~/cilium/installation$ export ROUTE_TABLE_ID=rtb-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 associate-route-table --subnet-id $SUBNET_ID --route-table-id $ROUTE_TABLE_ID
+    core@host:~/cilium/installation$ aws ec2 create-route --route-table-id $ROUTE_TABLE_ID --destination-cidr-block 0.0.0.0/0     --gateway-id $GATEWAY_ID
+    core@host:~/cilium/installation$ aws ec2 modify-vpc-attribute --vpc-id=$VPC_ID --enable-dns-support
+    core@host:~/cilium/installation$ aws ec2 modify-vpc-attribute --vpc-id=$VPC_ID --enable-dns-hostnames
+
+1.2 Configuring Key Pair and Security Group 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium/installation$ aws ec2 create-key-pair --key-name key --query 'KeyMaterial' --output text > key.pem
+    core@host:~/cilium/installation$ chmod 400 key.pem
+    core@host:~/cilium/installation$ aws ec2 create-security-group --group-name SG --description SG --vpc-id $VPC_ID
+    core@host:~/cilium/installation$ export SECURITY_GROUP_ID=sg-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 authorize-security-group-ingress --group-id $SECURITY_GROUP_ID --protocol tcp --port 22     --cidr x.x.x.x/8
+    core@host:~/cilium/installation$ aws ec2 authorize-security-group-ingress --group-id $SECURITY_GROUP_ID --protocol tcp --port 22     --cidr x.x.x.x/32
+    core@host:~/cilium/installation$ aws ec2 authorize-security-group-ingress --group-id $SECURITY_GROUP_ID --source-group     $SECURITY_GROUP_ID  --protocol all --port all
+
+1.3 Spinning up the VMs  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium/installation$ export IMAGE_ID=ami-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 describe-images --image-ids ami-xxxxxxxx
+
+
+1.3.1 CREATE MASTERS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+::
+
+    core@host:~/cilium/installation$ wget https://github.com/coreos/container-linux-config-transpiler/releases/download/v0.3.1/    ct-v0.3.1-x86_64-unknown-linux-gnu
+    core@host:~/cilium/installation$ chmod +x ct-v0.3.1-x86_64-unknown-linux-gnu
+
+::
+
+    core@host:~/cilium/installation$ vi master-config.yaml
```

Comment:
Consider pulling the comments out of the code block below and using as an introductory statement to the file creation.